src/sql/magic.py

```diff
@@ -619,6 +622,73 @@ def _persist_dataframe(
         display.message_success(f"Success! Persisted {table_name} to the database.")
 
 
+def find_toml_path():
+    """
+    Recursively finds an absolute path to pyproject.toml starting
+    from current directory
+    """
+    current = Path().resolve()
+    while not (current / "pyproject.toml").exists():
+        if current == current.parent:
+            return None
+
+        current = current.parent
+    display.message(f"Found pyproject.toml from '{current}'")
+
+    return str(current) + "/pyproject.toml"
+
+
+def get_configs(file_path):
+    """Returns saved configs from given file_path"""
+    with open(file_path, "r") as file:
+        content = file.read()
+        data = toml.loads(content)
+    try:
+        return data["tool"]["jupysql"]["SqlMagic"]
```

Comment:
can we generalize it? If it's too complicated, let's add it to the docstring that this is valid only for SqlMagic.
I mention it since `get_configs` and `get_config_variables`, and `load_configs` might be useful methods we utilize in the future.

Reply:
Agreed with @yafimvo

Reply:
- `get_configs` is now `get_user_configs` in `util.py` with a generalized parameter `section_names` to specify `["tool"]["jupysql"]["SqlMagic"]`
- `get_config_variables` is now `get_default_configs` in `util.py` with indicating it is for SqlMagic in its doc.
- `load_configs` is now `load_SqlMagic_configs` in `magic.py`